docs/get-started/using-with-react.md

```diff
@@ -7,63 +7,46 @@ render it as a child of another component, passing in your list of deck.gl
 layers as a property.
 
 ```jsx
-// Import React
+/// app.js
 import React, {Component} from 'react';
-import {render} from 'react-dom';
-
-// Import Mapbox via react-map-gl
 import MapGL from 'react-map-gl';
-
-// Import deck.gl and the layer you'd like to use
 import DeckGL, {LineLayer} from 'deck.gl';
 
-// Set your mapbox token here
-const MAPBOX_TOKEN = process.env.MAPBOX_ACCESS_TOKEN;
+// Set your mapbox access token here
+const MAPBOX_ACCESS_TOKEN = 'MAPBOX_ACCESS_TOKEN';
 
+// Viewport settings that is shared between mapbox and deck.gl
 const viewport = {
    width: 500,
    height: 500,
    longitude: -100,
    latitude: 40.7,
    zoom: 3,
-   maxZoom: 15,
    pitch: 0,
    bearing: 0
 }
 
-// The root React component
-class Root extends Component {
-  constructor(props) {
-    super(props);
-  }
+// Data to be used by the LineLayer
+const data = [
+  {sourcePosition: [-122.41669, 37.7853], targetPosition: [-122.41669, 37.781]}
+];
+
+export default class App extends Component {
 
   render() {
-    const {viewport, width, height} = this.state;
+
+    const layers = [
```

Comment:
I think we are doing users a disservice by constructing the layers outside of the JSX. It makes it less clear that layers are supposed to be created every render cycle, just like React elements.
Therefore I think the example should look like this:
```
<MapGL {...viewport} mapboxApiAccessToken={MAPBOX_ACCESS_TOKEN}>
  <DeckGL {...viewport} layers={[
    new LineLayer({id: 'line-layer', data})
  ]} />
</MapGL>   new LineLayer({id: 'line-layer', data})
```
In the next release we can support JSX syntax for layers, then this will be a non-issue.